kv/union_store.go

```diff
@@ -97,6 +97,7 @@ func (it invalidIterator) Value() []byte {
 
 func (it invalidIterator) Close() {}
 
+// lazyMemBuffer wraps a MemBuffer which is only inited when we modify it.
```

Comment:
-> lazyMemBuffer wraps a MemBuffer which is to be initialized only when it is modified.